de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/provider/RequirementsAttributeLabelProvider.java

```diff
@@ -177,6 +184,35 @@ public Image getColumnImage(Object object, int columnIndex) {
 
 		return super.getColumnImage(object, columnIndex);
 	}
+	
+	/**
+	 * Get the icon for the requirement status
+	 * @param statusValue the current status value
+	 * @return the image
+	 */
+	protected Image getStatusIcon(String statusValue) {
+		if (statusValue == null) {
+			return null;
+		}
+		String imagePath = ICON_STATUS_OPEN_PATH;

```

Comment:
Would it be possible to make the structure here a bit simple by extracting a getImagePath(statusValue) method?

Reply:
Not sure why it doesn't show outdated... I applyed your suggestion however... Although I don't really see a simplification here :D